src/payment/strategies/klarna/klarna-payment-strategy.spec.ts

```diff
@@ -118,12 +126,49 @@ describe('KlarnaPaymentStrategy', () => {
 
         it('loads widget', () => {
             expect(klarnaCredit.init).toHaveBeenCalledWith({ client_token: 'foo' });
+            expect(klarnaCredit.load)
+                .toHaveBeenCalledWith({ container: '#container' }, {}, expect.any(Function));
             expect(klarnaCredit.load).toHaveBeenCalledTimes(1);
         });
 
+        it('loads widget in EU', async () => {
+            store = createCheckoutStore(getCheckoutEUStoreState());
```

Comment:
Since this is only used once, there's no need to add the `getCheckoutEUStoreState` method. Just overwrite the part of the state that you care about: 

```suggestion
            store = createCheckoutStore({
            	...getCheckoutStoreState(),
            	billingAddress: { data: getEUBillingAddress(), errors: {}, state: {} },
            });
```